### DISCUSSION TITLE
Weapons should Match Searchlight kW usage

### DISCUSSION THREAD

**Discussion by Saevon** [FakeFishGames/Barotrauma#4731] [Ideas and suggestions]
Weapon's lights are identical to searchlights, you can toggle their lights off with wires if needed. So they should (by default) use up power like a searchlight until disabled.

This would be:
 * `min voltage`: 0.5
 * `power used`: 50

(I've changed this on my subs, but I think the default value, and existing subs should account for this too)

**Comment by Xeavas** [comment]
Would need to consider that weapon lights might be the only source of light to shoot enemies when in dire situations, the concern comes when it becomes a death spiral too often, so commenting here to note.

Edit : Possibly a dimmer light when unpowered to solve the mentioned concern.

**Reply by DreamerOfRain** [reply]
Also worth remembering that very often the sole source of power for guns are supercapacitators which only hold a measly 20kwm capacity. A constant light on will drain it very quickly.

**Reply by Saevon** [reply]
Would having: `Power_gun_in` and `Power_light_in` be doable?   I wonder
